client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-method-list.js

```diff
@@ -117,7 +117,7 @@ const ShippingZoneMethodList = ( {
 						{ translate( 'Method' ) }
 					</ListItemField>
 					<ListItemField className="shipping-zone__methods-column-summary">
-						{ translate( 'Details' ) }
+						{ translate( 'Cost' ) }
```

Comment:
This might not always be a cost.

> This column won't necessarily contain just the cost. For example, if free shipping is enabled for everyone, it will show "For everyone". In the designs, if we add WCS integration, it would show the enabled USPS services count.